v2/shared/keypress.tsx

```diff
@@ -104,6 +104,7 @@ export interface KeyFxnProps {
     action: KeyAction;
     combo?: boolean;
     target?: React.MutableRefObject<any>;
+    targets?: React.MutableRefObject<any>[];
```

Comment:
I prefer to combine the `target` and `targets` props into one. To maintain backwards compatibility, it can retain the name `target`, but be of type `React.MutableRefObject<any> || React.MutableRefObject<any>[]`. Then we can  check if this prop is an array with `if (target.length)`

Reply:
Hi @rbreeze , I've combined them.  keypress.tsx was updated to combine them.  See the bulk of the changes starting from line 132 to handle both situations.   I tested this with pressing escape on any opened sliding panel, as well as pressing the forward slash on the application list page to activate the search bar.